src/cow-react/utils/format.ts

```diff
@@ -191,3 +191,9 @@ export function truncateOnMaxDecimals(value: string, decimals: number): string {
   const regex = new RegExp(`(\\d*\\.\\d{${decimals}})\\d*`)
   return value.replace(regex, '$1')
 }
+
+const MAX_SYMBOL_LENGTH = 8
+
+export function formatSymbol(symbol: string | undefined): string | undefined {
```

Comment:
It works great, however, would be cool to create a component for this.
Like:
```
function TokenSymbol({token}){
    return <span title={token.symbol}>{formatSymbol(token.symbol)}</span>
}
```

Reply:
Great idea, I've updated it